packages/phishing-controller/src/PhishingDetector.test.ts

```diff
@@ -1331,6 +1331,90 @@ describe('PhishingDetector', () => {
       );
     });
 
+    it('should return allowlist if URL is in the allowlist', async () => {
```

Comment:
Can we make this test without the URL existing on the `c2DomainBlocklist` and a separate test that checks that the `allowlist` takes precedence over the `c2DomainBlocklist`?